packages/grafana-ui/src/components/GraphNG/utils.ts

```diff
@@ -237,6 +254,25 @@ export function preparePlotConfigBuilder(
       }
     }
   }
+
+  // Always publish events
+  const syncMode = sync?.sync ?? DashboardCursorSync.Off;
+  const cursorSync: uPlot.Cursor.Sync = {
+    key: uPlotGlobalEvents.globalKey, // + '/' + xScaleKey + '/' + yScaleKey,
+    filters: {
+      pub: uPlotGlobalEvents.filter,
+    },
+    setSeries: syncMode === DashboardCursorSync.Tooltip,
+    scales: ['x', 'y'], //TODO: [xScaleKey, yScaleKey],
```

Comment:
I'm not sure the best approach here -- using x/y it works OK (ie does not crash!) but is syncing on pixels, not units.  I think we want to select/hover keeping the same units.

When you put in the scaleKeys above, the behavior is great, but there are internal errors sent to the log when the scale does not exist